src/js/messaging/containers/Thread.jsx

```diff
@@ -106,7 +112,7 @@ class Thread extends React.Component {
       header = (
         <ThreadHeader
             currentMessageNumber={currentIndex + 1}
-            folders={this.props.folders}
+            folders={folders}
```

Comment:
Could these be named something like `moveToFolders` to make it clear what they're for?